index2.html

```diff
@@ -0,0 +1,360 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style2.css" />
+
+    <title>David Underwood Home Page</title>
+  </head>
+  <body>
+    <header class="header1">
+      <div class="header">
+        <img class="cell-info" src="Light.png" alt="" />
+      </div>
+
+      <div class="bottom-part">
+        <div class="welcome">
+          <a href="https://github.com/david63011">Welcome</a>
+        </div>
+        <img class="union" src="Union.png" alt="Hamburger" />
+      </div>
+    </header>
+
+    <section class="main-part">
+      <img class="background-photo1" src="background.png" alt="" />
+      <img class="background-photo2" src="desktop.svg" alt="" />
+      <h2 class="porto">Portoflio</h2>
+      <h2 class="about-top">About</h2>
+      <h2 class="con">Contact</h2>
+
+      <div class="greeting">
+        <h3 class="hi">Hey there. I'm</h3>
+        <h3 class="first-name">David</h3>
+        <h3 class="job">I am a software</h3>
+        <h3 class="job-info">Developer</h3>
+      </div>
+      <div class="greeting2">
+        <h3 class="hi-alex">Hey there. I'm David.</h3>
+        <h3 class="job-two">I am a software developer</h3>
+      </div>
+      <div class="information-box">
+        <p class="paragraph">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+    </section>
+    <div class="icon-row">
+      <div class="tweet-connect">
+        <a href="https://twitter.com/david630"
+          ><img class="tweet" src="twitter.svg" alt="twitter picture"
+        /></a>
+      </div>
+
+      <img class="med" src="medium.svg" alt="medium picture" />
+      <div>
+        <a href="https://github.com/david63011"
+          ><img class="git" src="github.svg" alt="git picture"
+        /></a>
+      </div>
+
+      <img class="angel" src="angellist.svg" alt="angelist picture" />
+    </div>
+
+    <section class="top-one">
+      <h1 class="work">My Recent</h1>
+      <h1 class="work-2">Works</h1>
+    </section>
+
+    <img class="Indicator" src="Indicator.png" alt="" />
+
+    <section class="grid-containers">
+      <div class="container">
+        <div class="column1"></div>
+        <div class="column2">
+          <div class="project-box">
+            <h1 class="project-name1">Multi-Post Stories</h1>
+            <h1 class="project-name-two">Gain+Glory</h1>
+          </div>
+
+          <div class="languages">
+            <ul>
+              <li class="ruby">Ruby on Rails</li>
+              <li class="css">css</li>
+              <li class="js">JavScript</li>
+              <li class="html">HTML</li>
+            </ul>
+          </div>
+          <button type="submit" class="see-project-btn">See Project</button>
+        </div>
```

Comment:
- [ ] Great job 👍  . Kindly fix the color contrast issue of all the buttons in your project section 🙏 

![image](https://user-images.githubusercontent.com/74814780/188690913-e8488146-56c3-48aa-ac0d-e5836abb2940.png)